src/xdocs/config_naming.xml

```diff
@@ -207,31 +216,31 @@
           </tr>
           <tr>
             <td>format</td>
-            <td>Specifies valid identifiers.</td>
+            <td>Specify valid identifiers.</td>
             <td><a href="property_types.html#regexp">Regular Expression</a></td>
-            <td><code>"^Abstract.+$"</code></td>
+            <td>"^Abstract.+$"</td>
             <td>3.2</td>
           </tr>
           <tr>
             <td>ignoreModifier</td>
             <td>
-              Controls whether to ignore checking for the
+              Control whether to ignore checking for the
               <code>abstract</code> modifier on classes that match the
               name.
             </td>
             <td><a href="property_types.html#boolean">Boolean</a></td>
-            <td><code>false</code></td>
+            <td>false</td>
             <td>5.3</td>
           </tr>
           <tr>
             <td>ignoreName</td>
             <td>
-              Controls whether to ignore checking the name. Realistically
+              Control whether to ignore checking the name. Realistically
               only useful if using the check to identify that match name
-              and do not have the <code>abstract</code> modifier.  name.
+              and do not have the <code>abstract</code> modifier.
             </td>
             <td><a href="property_types.html#boolean">Boolean</a></td>
-            <td><code>false</code></td>
+            <td>false</td>
```

Comment:
Can we keep code tag ? It looks good in result html.

Reply:
Done.